### PR TITLE
Passed link target prop up for markdowns

### PR DESCRIPTION
**Pull request by YextSLu**
J=CLIP-1293
Test=auto/manual

Ran jest tests locally and used `test-site` to manually change/pass link target types and opened the links in the chatbot.

### CODE REVIEW

File: src/components/ChatPanel.tsx [hunk 1]
```diff
@@ -75,6 +75,8 @@ export interface ChatPanelProps
    * can click on instead of typing their own.
    */
   messageSuggestions?: string[];
+  /** Link target open behavior on click. */
```
Comment (yen-tt) [inline src/components/ChatPanel.tsx:78]: nit: can you specify the default behavior in the comment as well?

**Review by yen-tt — COMMENTED** [review]
lgtm beside a small comment!

File: src/components/ChatPanel.tsx (hunk 1 above)
Comment (YextSLu) [reply src/components/ChatPanel.tsx]: Fixed!

File: tests/components/ChatPanel.test.tsx [hunk 2]
```diff
@@ -149,3 +149,29 @@ it("executes onLinkClick if provided", async () => {
...
+    conversation: {
+      messages: [{ text: "Test [msg link](msglink)" }],
+      isLoading: false,
+      canSendMessage: true,
+    },
+  });
+
+  render(<ChatPanel linkTarget={"_blank"} />);
+  expect(screen.getByText('msg link')).toHaveAttribute('target', "_blank");
+});
```
Comment (nbramblett) [inline tests/components/ChatPanel.test.tsx:177]: Can you add an additional test that ensures the target is still _blank when _no_ value is provided?

Comment (YextSLu) [reply tests/components/ChatPanel.test.tsx]: Added!

**Pull request merged**